AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -293,6 +282,27 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
 print("Using %s as the reference antenna "%(ant_list[0]))
 h5.select(compscans='interferometric_pointing',ants=ant_list)
 
+if len(opts.channel_mask)>0:
```

Comment:
The mask -> rfi_static_flags extractor seems like it should be a function. It only depends on `opts.channel_mask`, `h5` and `chunks`.